requirements/base.txt

```diff
@@ -1,9 +1,11 @@
 #
-# This file is autogenerated by pip-compile with Python 3.8
+# This file is autogenerated by pip-compile with Python 3.10
```

Comment:
didn't you call out an issue on compiling with 3.10? assuming you'll hit that when you merge up?